openassessment/templates/openassessmentblock/peer/oa_peer_waiting.html

```diff
@@ -2,7 +2,7 @@
 {% load i18n %}
 
 {% block list_item %}
-<li id="openassessment__peer-assessment__{{ xblock_id }}" class="openassessment__steps__step step--peer-assessment is--showing ui-slidable__container" tabindex="-1">
+<li id="openassessment__peer-assessment__{{ xblock_id }}" class="openassessment__steps__step step--peer-assessment is--incomplete is--showing ui-slidable__container" tabindex="-1">
```

Comment:
Waiting for peer assessments to grade will now render with the yellowish "warning" color, which is mapped to the pre-existing is--incomplete class.